Week1/homework/style.css

```diff
@@ -1 +1,29 @@
-/* add your styling here */
\ No newline at end of file
+/* add your styling here */
+html {
```

Comment:
I like the creativity you put in the design!

Reply:
Thank you